desktop/src/com/mygdx/game/client/network/DesktopEntityConfigHandler.java

```diff
@@ -4,38 +4,43 @@
 import com.github.czyzby.websocket.WebSocket;
 import com.mygdx.game.assets.GameConfigAssets;
 import com.mygdx.game.client.ecs.entityfactory.ModelInstanceCompSetter;
+import com.mygdx.game.client.ecs.entityfactory.Setter;
 import com.mygdx.game.client_core.network.ComponentMessageListener;
-import com.mygdx.game.config.ModelConfig;
 import com.mygdx.game.core.ecs.component.EntityConfigId;
 import lombok.extern.java.Log;
 
 import javax.inject.Inject;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import static com.github.czyzby.websocket.WebSocketListener.FULLY_HANDLED;
 import static com.github.czyzby.websocket.WebSocketListener.NOT_HANDLED;
 
 @Log
 public class DesktopEntityConfigHandler implements ComponentMessageListener.Handler {
 
   private final GameConfigAssets assets;
-  private final ModelInstanceCompSetter modelInstanceCompSetter;
+  private final List<Setter> setterList = new ArrayList<>();
 
   @Inject
   public DesktopEntityConfigHandler(
       GameConfigAssets assets,
       ModelInstanceCompSetter modelInstanceCompSetter
   ) {
     this.assets = assets;
-    this.modelInstanceCompSetter = modelInstanceCompSetter;
+    this.setterList.add(modelInstanceCompSetter);
   }
 
   @Override
   public boolean handle(WebSocket webSocket, int worldEntity, Component component) {
     var entityConfigId = ((EntityConfigId) component).getId();
     var config = assets.getGameConfigs().get(entityConfigId);
-    if (config instanceof ModelConfig modelConfig) {
-      modelInstanceCompSetter.set(modelConfig, worldEntity);
-      return FULLY_HANDLED;
+    log.info("MY THREAD: " + Thread.currentThread().getName() + " THREAD_ID: " + Thread.currentThread().getId());
```

Comment:
Mam wątpliwości co do tego, co jeśli będę chciał, żeby dla jednego entity dwa settery miały coś zrobić? Pierwszy zwróci FULLY_HANDLED i nie będzie mogło wykonać drugiego, zastanów się nad tym :P

Reply:
a faktycznie, to zmienię później